doc/src/site/sphinx/deployment/kubernetes.rst

```diff
@@ -246,3 +246,120 @@ Dynamic allocation must be disabled in Spark.
             Rscript --default-packages=methods,utils batch.R
 
         Note: In case of RSparkling, SparklyR automatically sets the Spark deployment mode and it is not possible to specify it.
+
+External Backend
+~~~~~~~~~~~~~~~~
+
+Sparkling Water External backend can be also used in Kubernetes. First, we need to start external H2O backend on
+Kubernetes. This can be achieved by the following steps:
+
+1. Create Headless Service for H2O node discovery
+
+.. code:: bash
+
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: h2o-service
+    spec:
+      type: ClusterIP
+      clusterIP: None
+      selector:
+        app: h2o-k8s
+      ports:
+      - protocol: TCP
+        port: 54321
+    EOF
+
+2. Deploy external H2O cluster of size 2 as:
+
+.. code:: bash
+
+    cat <<EOF | kubectl apply -f -
+    apiVersion: apps/v1
+    kind: StatefulSet
+    metadata:
+      name: h2o-stateful-set
+      namespace: default
+    spec:
+      serviceName: h2o-service
+      replicas: 2
+      selector:
+        matchLabels:
+          app: h2o-k8s
+      template:
+        metadata:
+          labels:
+            app: h2o-k8s
+        spec:
+          terminationGracePeriodSeconds: 10
+          containers:
+            - name: h2o-k8s
+              image: 'h2oai/sparkling-water-external-backend:SUBST_SW_VERSION'
+              resources:
+                requests:
+                  memory: "2Gi"
+              ports:
+                - containerPort: 54321
+                  protocol: TCP
+              readinessProbe:
+                httpGet:
+                  path: /kubernetes/isLeaderNode
+                  port: 8081
+                initialDelaySeconds: 5
+                periodSeconds: 5
+                failureThreshold: 1
+              env:
+              - name: H2O_KUBERNETES_SERVICE_DNS
+                value: h2o-service.default.svc.cluster.local
+              - name: H2O_NODE_LOOKUP_TIMEOUT
+                value: '180'
+              - name: H2O_NODE_EXPECTED_COUNT
+                value: '2'
+              - name: H2O_KUBERNETES_API_PORT
+                value: '8081'
+    EOF
+
+It is important the image to be `h2oai/sparkling-water-external-backend:SUBST_SW_VERSION` and not the base H2O image.
+Sparkling Water enhances the H2O image with additional dependencies.
+
+For more information about H2O on Kubernetes, please read the
+`H2O on Kubernetes Blog Post <https://www.h2o.ai/blog/running-h2o-cluster-on-a-kubernetes-cluster/>`__.
```

Comment:
Are blog posts at H2O being updated? If not, isn't there a relevant documentation for H2O's Kubernetes support?

Reply:
We probably don't update them. Good point, we have documentation, will point to that